src/Providers/TransfererServiceProvider.php

```diff
@@ -56,9 +56,7 @@ public function register()
     /**
      * Bootstrap the application services.
      */
-    public function boot()
-    {
-    }
+    public function boot() {}
```

Comment:
Need to update `php-cs-fixer.dist.php` to our latest team configuration and revert this edit.